### ISSUE TITLE
Orange Tv GO lista m3u

### ISSUE THREAD

**Issue by pbaloo** [mbebe/blomqvist#889]
Jak wyeksportować z wtyczki w KODI listę do m3u?

**Comment by shafau** [comment]
Dokładnie, przydałaby się taka możliwość, bo tworzenie listy krok po kroku na podstawie ulubionych jest dość uciążliwe...

**Comment by angustPL** [comment]
dołączam
